AntennaTracker/control_auto.cpp

```diff
@@ -60,6 +60,12 @@ void Tracker::calc_angle_error(float pitch, float yaw, bool direction_reversed)
     convert_ef_to_bf(ef_pitch_angle_error, ef_yaw_angle_error, bf_pitch_err, bf_yaw_err);
     nav_status.angle_error_pitch = bf_pitch_err;
     nav_status.angle_error_yaw = bf_yaw_err;
+
+    // set actual and desired for logging
+    g.pidPitch2Srv.set_desired_rate(pitch * 0.01);
+    g.pidPitch2Srv.set_actual_rate(ahrs_pitch * 0.01);
```

Comment:
I'm sure you're aware, but these are aren't angular rates, they're angles.  Probably shows the methods on the object are misnamed.

Reply:
Any update on this?